### DISCUSSION TITLE
Historical data JSON

### DISCUSSION THREAD

**Discussion by MarcinRawski** [RavenSystem/esp-homekit-devices#1654] [Q&A] [answered]
Hi,
I have problem with historical data service setup. How it should look?
My JSON  `{ "c": { "l": 13, "ntp" : "192.168.1.1", "tz" : "CET-1CEST-2,M3.5.0/2,M10.5.0/3", "b": [ { "g": 0, "t": 5 } ] }, "a": [ { "0": { "r": [ { "g": 12 }, { "g": 13, "v": 0 } ] }, "1": { "r": [ { "g": 12, "v": 1 }, { "g": 13, "v": 1 } ] }, "b": [ { "g": 0 } ], "s": 0 }, { "t": 22, "g": 3, "n": 3, "b": [], "j": 15, "z": 0.5 }, { "t": 5, "f0": [ { "g": 1, "t": 0 } ], "f1": [ { "g": 1 } ], "s": 0, "es": [{ "t": 95, "h": [ 5, 0, 5 ] }] } ] }`

**Comment by pawelosin** [comment, marked as answer]
Cześć, jeżeli nie znalazłeś jeszcze rozwiązania, to musisz zmienić konfigurację historii na: "es": [{ "t": 95, "h": [ 3, 0, 5 ] }]

**Reply by MarcinRawski** [reply]
Dzięki. Pomogło, teraz w aplikacji widzę serwis danych historycznych, niestety cały czas status : total records : 0. Możesz podpowiedzieć co mogę jeszcze mieć skopane ?

**Reply by MarcinRawski** [reply]
Działa. Zmieniłem serwer czasu.
